### PR TITLE
Release 2.9.13-beta1

### PR DESCRIPTION
**Pull request by sergiou87**
## Description
Looking for the PR for the upcoming first beta of the v2.9.13 series? Well, you've just found it, congratulations!

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release
- [x] Verify that all feature flags are flipped appropriately
- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.9.13-beta1": [
+      "[New] Add support for notifications of pull request reviews - #14175",
+      "[Fixed] Pull requests adhere to temporal laws again - #14238",
```
Comment (sergiou87) [inline changelog.json:5]: Not sure if there is precedent of funny release notes like this, but I personally love it 😂

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+      "[Fixed] Fix repository group header overflow when text is too long - #14233. Thanks @tsvetilian-ty!",
+      "[Fixed] Clone dialog \"Choose\" button uses an open dialog for directory selection on macOS - #12812",
```
Comment (tidy-dev) [inline changelog.json]: :D - my bad on writing the original note.. but is windows that is changing not mac.

```suggestion
      "[Fixed] Clone dialog \"Choose\" button uses an open dialog for directory selection on windows - #12812",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+      "[Improved] Add link to beta download page in about dialog - #14237"
```
Comment (say25) [inline changelog.json]:
```suggestion
```
Technically this doesn’t show in the betas but I don’t recall how release notes do or don’t build on each other 

Also this should really be a Added vs Improve

```suggestion
      "[Added] Link to beta download page in about dialog - #14237"
```

Comment (sergiou87) [reply changelog.json]: You're right it shouldn't be in these release notes at all, but I think `Improved` is the right category according to https://github.com/desktop/desktop/blob/241c4a2cfda89dc60d2ee7a094657ad349de7274/docs/process/writing-release-notes.md/#L129-L134

I'll remove it, thanks!! ❤️

**Review by tidy-dev — CHANGES_REQUESTED** [review]

File: changelog.json (hunk 2 above)
Comment (tidy-dev) [reply changelog.json]: .... So I thought I left this comment yesterday when the pr was opened.. but it was still pending. :face-palm:

**Comment by bayleev33me** [comment]
Ok having a bad network trying to open site I have even I have data

David bayleev

File: changelog.json (hunk 2 above)
Comment (sergiou87) [reply changelog.json]: Fixed in [ee57829](https://github.com/desktop/desktop/pull/14245/commits/ee5782916f73a517a71762b9ed1909873fbfcbd5)!

**Review by tidy-dev — APPROVED** [review]
✨

**Pull request merged**